cowprotocol/cowswap
# Limit orders/faq

# Summary

This PR adds a limit-order FAQ page.

![Снимак екрана из „2022-12-09 13-03-55“](https://user-images.githubusercontent.com/34926005/206698602-8d22602b-fb3e-4889-9788-15ef15eaec23.png)

Also adds links in header dropdown and sidenav 
![Снимак екрана из „2022-12-09 13-07-21“](https://user-images.githubusercontent.com/34926005/206699257-e2adbb02-ec2a-42bb-832f-2a2fc399a435.png)

# To test
- go to `/faq/limit-order`
- make sure everything works fine
